realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -215,10 +217,6 @@ public static SharedRealm getInstance(RealmConfiguration config, RealmNotifier r
         }
     }
 
-    long getNativePtr() {
```

Comment:
it looks better to be together with `getNativeFinalizerPtr()` and close to `close()`. :)